crowd_anki/meta.json

```diff
@@ -0,0 +1 @@
+{"config": {"automated_snapshot": false, "snapshot_root_decks": [], "export_deck_sort_reversed": false, "export_deck_sort_method": ["none"], "snapshot_path": "/home/jmo/Development/CrowdAnki/crowd_anki/user_files"}}
```

Comment:
prettify this one? (e.g. newlines, indentations/etc)

also, you've commited your snapshot path

Reply:
also, can't comment there, but `zzzNoteTypeExamples` probably wasn't meant to be commited either?

Reply:
Yes and yes! :sweat_smile:

Reply:
Alright, deleted my personal files. But the meta.json is just the local copy of the settings, it gets made everytime the config saves. As it's not in your own master branch and there is no use in having it in source control I deleted it and added it to the gitignore :+1: